src/hooks/useDrag.tsx

```diff
@@ -184,6 +184,31 @@ export const useDrag: OnDragEndResponseWrapper = (
   return updateHomepageState(result, homepageState)
 }
 
+type UpdateHomepageType =
+  | "editor"
+  | "dropdownelem"
+  | "highlight"
+  | "announcement"
+  | `textcardcard-${number}`
+
+const isUpdateHomepageType = (value: any): value is UpdateHomepageType => {
+  if (typeof value === "string" && value.startsWith("textcardcard-")) {
+    const valArr = value.split("-")
+    const number = valArr[1]
+    return valArr.length === 2 && !Number.isNaN(Number(number))
```

Comment:
Nit: I think can just use `isNaN` directly without `Number.isNaN`?

Reply:
Is `!Number.isNaN(Number(number))` to assert if its a number?

In the case where value is just `textcardcard-`, split gives `['textcardcard', '']`

`!Number.isNaN(Number(number))` evaluates to true here

Reply:
Eslint prefers`Number.isNaN` because of the implicit coercion inside `isNaN`! I opted to use Number.isNan due to this

Good catch for the empty string, fixed in https://github.com/isomerpages/isomercms-frontend/pull/1527/commits/4cf4b018a4197a4a9fce80138f35577911cd81d0